CONTRIBUTING.md

```diff
@@ -15,6 +15,58 @@ You generally only need to submit a CLA once, so if you've already submitted one
 (even if it was for a different project), you probably don't need to do it
 again.
 
+## Tests
+
+### Unit Tests
+
+Unit tests can be run locally with minimal fuss.
+
+**Requirements**
+
+* [Node.js](https://nodejs.org/en/download/).
```

Comment:
If we upgrade to a newer version of [`rules_nodejs`](https://github.com/bazelbuild/rules_nodejs) can we remove node as a system dependency? The newer version appear to pull in node and yarn in a toolchain-like way.